transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/installments.cshtml

```diff
@@ -73,10 +73,10 @@
             <div class="card">
               <div class="card-body">
                 <h6 class="card-title">Formulario confirmación con cuotas</h6>
-                <form action="@ViewBag.commit-endpoint")}" method="POST">
+                <form action="@ViewBag.CommitEndpoint" method="POST">
                   <div class="mb-3">
                       <label for="token" class="form-label">Token</label>
-                      <input type="text" class="form-control" id="token" name="token" value="@ViewBag.Token">
+                      <input type="text" class="form-control" id="token_ws" name="token_ws" value="@ViewBag.TokenWs">
```

Comment:
This is necessary?

Reply:
No, changes were made to the controller and changes were made to the view.